src/ess/polarization/base.py

```diff
@@ -307,14 +307,25 @@ def he3_opacity_from_beam_data(
     direct_beam: DirectBeamNoCell,
     direct_beam_cell: He3DirectBeam[Cell, Depolarized],
 ) -> He3Opacity[Cell]:
+    
+    def Intensity_direct_beam_cell(time, opacity_cell):
+        return direct_beam*transmission_empty_glass*sc.exp(-opacity_cell)
+    
+    popt, pcov = sc.curve_fit(['time'], reduce_dims = ['wavelength'], Intensity_direct_beam_cell, direct_beam_cell)
+    # Q: correct to use 'time' on x-axis?
+    # expected output: popt = opacity = single parameter for each cell, wavelength independent
```

Comment:
I don't get how you plan to simply drop the wavelength dependence from the data / equations. The direct beam data is wavelength dependent, so I thought this must be taken into account in the fit function. Can you write this down clearly "on paper" (i.e., as part of #16) before moving to implementation?